### DISCUSSION TITLE
How to move after::pseudo-selector behind parent?

### DISCUSSION THREAD

**Discussion by pexeixv** [tailwindlabs/tailwindcss#11569] [Help] [answered]
Trying to make a button like this
![image](https://github.com/tailwindlabs/tailwindcss/assets/68476117/5a679916-cb64-438e-8726-9790a5a10cde)

This is my code
```
<section class="grid min-h-screen place-items-center">
  <a class="relative z-20 inline-block border-2 border-emerald-200 bg-emerald-800 px-6 py-2 text-emerald-200 before:absolute before:left-4 before:top-4 before:h-full before:w-full before:bg-emerald-200 before:content-[''] after:-z-10"> Text </a>
</section>
```

But the button looks like this instead 
![image](https://github.com/tailwindlabs/tailwindcss/assets/68476117/3fc8b876-376b-4a68-b2b2-4703542f6415)

Tailwind Playground: https://play.tailwindcss.com/kguEEASuKR

**Comment by wongjn** [comment, marked as answer]
It seems like you're using the `::before` pseudo element to construct the visual effect, not the `::after` pseudo element that you mention in your title. Furthermore, since the `<a>` element is a stacking context, the `::before` can never be behind the background of its own stacking context.

Instead, please find two alternative techniques to achieve the visual effect in [this Tailwind Play](https://play.tailwindcss.com/454QHdi7wQ).

**Comment by pexeixv** [comment]
Thanks a lot!
